src/repl/repl.cpp

```
@@ -101,7 +101,7 @@ static bool read_line(std::istream& in, std::ostream& /*out*/, std::string& line
   }
   // Build prompt: "nick@provider:model> " or "nick@provider> " or "nick> " or "> "
   std::string label = build_prompt_label(nick, provider, model);
-  std::string prompt_str = color ? tui::active_theme().prompt.ansi() + label + Style::reset() : label;
+  std::string prompt_str = color ? tui::active_theme().prompt.ansi() + label + ThemeStyle::reset() : label;
```

Clang (14.0.6)

[warning] variable 'prompt_str' is not initialized

(cppcoreguidelines-init-variables)